src/modules/caen/CaenCounters.h

```diff
@@ -21,12 +22,12 @@ struct CaenCounters {
   // Processing Counters - accessed in processing thread
 
   // System counters
-  int64_t FifoSeqErrors;
-  int64_t ProcessingIdle;
+  int64_t FifoSeqErrors{};
```

Comment:
Could we initialize them to 0. I'm not sure {} probably does it, but for readability it would be more straight

Reply:
probably a fine idea.